src/diffusers/pipelines/stable_diffusion_xl/pipeline_stable_diffusion_xl_inpaint.py

```diff
@@ -939,6 +939,17 @@ def _encode_vae_image(self, image: torch.Tensor, generator: torch.Generator):
         else:
             image_latents = retrieve_latents(self.vae.encode(image), generator=generator)
 
+        has_latents_mean = hasattr(self.vae.config, "latents_mean") and self.vae.config.latents_mean is not None
```

Comment:
why do we need it here?

Reply:
I think this would be needed because here we are encoding the input image. This is what we do in the SDXL DreamBooth LoRA training script.

Reply:
But I agree that #7132 tackles it from a comprehensive perspective.

Reply:
ok I re-opene it and I'm happy to be proven wrong
once we merge #7132, you can update the branch, and you can then test it out and show some results with and without the normalization.

Reply:
Works for me!